jenkins/jenkins-install.sh

```diff
@@ -43,7 +47,10 @@ else
     port=8080
 if
 
+echo "Configuring port to be used for jenkins service..."
```

Comment:
Default port is fine so we don't need the ability to set port and less code is better (less potential bugs).